src/Symfony/Component/HttpFoundation/Tests/ResponseTest.php

```diff
@@ -353,9 +353,8 @@ public function testGetMaxAge()
         $this->assertEquals(3600, $response->getMaxAge(), '->getMaxAge() falls back to Expires when no max-age or s-maxage directive present');
 
         $response = new Response();
-        $response->headers->set('Cache-Control', 'must-revalidate');
         $response->headers->set('Expires', -1);
-        $this->assertLessThanOrEqual(time() - 2 * 86400, $response->getExpires()->format('U'));
+        $this->assertSame(0, $response->getMaxAge());
```

Comment:
The test was for `getMaxAge()` however in this case it was testing `getExpires()`, which seemed wrong.